Makefile

```diff
@@ -157,4 +157,3 @@ clickhouse-monitor:
 clickhouse-monitor-plugin:
 	@mkdir -p $(BINDIR)
 	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/theia/plugins/clickhouse-monitor
-	
```

Comment:
new line

Reply:
I think it means the old file has no newline at end of file, as the last line have 2 spaces on the final line. I just removed that 2 spaces in this commit.